### ISSUE TITLE
livestreaming from the phone app

### ISSUE THREAD

**Issue by test2a** [stom79/TubeLab#100]
### Issue

this should be put on the backburner but livestreaming is coming to peertube very soon. i dont think there is any foss android streaming app that can stream to peertube.

i have found there are libraries on github that will enable this feature in app. maybe we can incorporate them here. this would mean if you have a willing instance, anyone can start streaming. much easier for democratization for the community than having to have a pc and stuff.

https://github.com/pedroSG94/rtmp-rtsp-stream-client-java

https://github.com/search?p=2&q=android+stream&type=Repositories

### Steps for reproducing the issue

---

Instance:

- [ ] TubeLab
- [ ] TubeAcad

Version of TubeLab:

Android version:

**Comment by test2a** [comment]
probably keep this for major version update but this should be the last of the big features when there are more pressing things

**Comment by stom79** [comment]
Viewing live streams already work. I need to put a dedicated label "Live" on videos. But making live through the app has to be done. I guess that was the purpose of that issue.

**Comment by test2a** [comment]
Yes. Both actually. I think there were discussions on having a separate live tab. That is not live yet but what I was thinking, there are a lot of "rtmp" streaming apps on play store. 
Tubelab could be used if it gets rtmp streaming to need only one app for all peertube content creation and consumption
